src/main/java/uk/gov/hmcts/darts/arm/mapper/impl/AnnotationArchiveRecordMapperImpl.java

```diff
@@ -306,7 +306,15 @@ private UploadNewFileRecordMetadata createUploadNewFileRecordMetadata(Annotation
         UploadNewFileRecordMetadata uploadNewFileRecordMetadata = new UploadNewFileRecordMetadata();
         uploadNewFileRecordMetadata.setPublisher(armDataManagementConfiguration.getPublisher());
         uploadNewFileRecordMetadata.setDzFilename(rawFilename);
-        uploadNewFileRecordMetadata.setFileTag(annotationDocument.getFileType());
+        uploadNewFileRecordMetadata.setFileTag(getFileTag(annotationDocument.getFileName()));
         return uploadNewFileRecordMetadata;
     }
+
+    private String getFileTag(String filename) {
+        String fileTag = null;
+        if (nonNull(filename) && filename.contains(".") && filename.lastIndexOf(".") + 1 < filename.length()) {
+            fileTag = filename.substring(filename.lastIndexOf(".") + 1);
+        }
+        return fileTag;
+    }
```

Comment:
Per Richard's earlier comment, if all we need is the extension then [FilenameUtils](https://commons.apache.org/proper/commons-io/apidocs/org/apache/commons/io/FilenameUtils.html#getExtension(java.lang.String)) will be a more robust alternative, and will allow you to eliminate this private method, and the other two identical occurences of it in:

- `src/main/java/uk/gov/hmcts/darts/arm/mapper/impl/CaseArchiveRecordMapperImpl.java`
- `src/main/java/uk/gov/hmcts/darts/arm/mapper/impl/TranscriptionArchiveRecordMapperImpl.java`